ui/ducks/index.js

```diff
@@ -17,7 +17,7 @@ export default combineReducers({
   activeTab: (s) => (s === undefined ? null : s),
   metamask: metamaskReducer,
   appState: appStateReducer,
-  ENS: ensReducer,
+  domainState: ensReducer,
```

Comment:
Any ideas for a more pithy name for this? domainState isn't really descriptive of what this is doing and it only matches the pattern of 'appState' which isn't really our most pristine example of reducers/state

Reply:
yeah we weren't sure of a good name here either - it definitely is used to hold the "state" of resolved domains, that's why we went with appState - would something like resolvedDomain be better?

Reply:
I like it - you don't think folks might get confused with traditional DNS?

Reply:
I guess end users wouldn't see it

Reply:
Probably, lol thats why I was asking if it is wrong. I feel 'domain' and 'domainState' are equally confusing because MetaMask (at least in the extension) runs in the context of the browser and there is always a 'domain' even if that domain is the chrome extension url. I do not want it to be confused with the state of the 'current' domain versus the state of the domain resolution in the send flow (and elsewhere it is used now and in the future)

Reply:
Another option is something like addressResolution - we're changing it to DNS for the time being but are open to other options